releases/R2023b/README.md

```diff
@@ -43,10 +42,15 @@ To deploy MATLAB Production Server onto an existing VPC, specify these additiona
 
 | Parameter Name | Value |
 |--------------- | ----- |
-| Existing VPC ID | ID of your existing VPC. |
+||**Network License Manager**|
+| **Deploy License Server** | Specify whether you want to deploy the Network License Manager for MATLAB. This parameter is available only if you use the deployment template for an existing VPC. <p>You can deploy a license server only if your solution uses public IP addresses. If your solution uses private IP addresses, you must separately deploy a license server in a public subnet.</p><p>If you are using an existing VPC, see the [Use Existing License Server in Existing VPC](#use-existing-license-server-in-existing-vpc) section.</p> |
+|**IP Address and port number of Existing Network License Manager**|<p>Specify the port number and private DNS name or private IP address of the network license manager that has already been deployed to the existing VPC. Specify it in the format <port>@<privateDNSname>, for example, 27000@ip-172-30-1-89.ec2.internal or 27000@172.30.1.89. By default, the license manager uses port 27000. Leave this field blank if you are deploying a new network license manager.</p>|
```

Comment:
You can mention that this is an optional parameter. You can also remove the hyperlink from 27000@ip-172-30-1-89.ec2.internal.

Reply:
Not sure why it's showing as a hyperlink.

Reply:
You can use monospace font using backticks for both the port@DNS name examples.